frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -200,6 +203,19 @@ const GraphicalSelector = () => {
     }
   };
 
+  const zoomIn = () => {
+    if (!graph) return;
+    const zoom = graph.getZoom();
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    zoom >= 1 ? graph.zoomTo(zoom + 0.5) : graph.zoomTo(zoom + 0.1);
+  };
+  const zoomOut = () => {
+    if (!graph) return;
+    const zoom = graph.getZoom();
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    zoom >= 1 ? graph.zoomTo(zoom - 0.5) : graph.zoomTo(zoom - 0.1);
```

Comment:
would be better to define `0.5` as a constant above and reuse that variable?

Reply:
yep zoomRatio is the new variable